check-your-phone.md

```diff
@@ -0,0 +1 @@
+You look up and suddenly, there is a woman in front of you. She tells you to get off your phone, because "You're on vacation in Paris!"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.